src/vdom/builder.rs

```diff
@@ -183,4 +180,10 @@ impl<Ms, Mdl, ElC: View<Ms> + 'static, GMs: 'static> Builder<Ms, Mdl, ElC, GMs>
 
         app
     }
+
+    /// Start the app; wraps `.finish()` and `.start()`.
```

Comment:
I think that it should be:
```rust
/// Start the app; wraps `.finish()` and `.run()`.
```
But I suggest to change it to:
```rust
/// Build and run the app.
```
and mark `finish` and `run` as deprecated.

---

I was googling a little bit to make sure that we don't make mistake with `build_and_run` (vs e.g. `start`) - and I can confirm that `build` is the most used final method for builders and it's used also in https://github.com/colin-kiegel/rust-derive-builder, so I still think that it's a good idea.
`build_and_run` vs `build_and_start` - `start` sounds better for application but our old API has method `run` - I don't have preference here.

Reply:
`build_and_start` works.